dask_planner/src/sql/optimizer.rs

```diff
@@ -122,14 +122,15 @@ mod tests {
         AND (cast('2002-05-08' as date) + interval '5 days')\
     )";
         let plan = test_sql(sql)?;
-        let expected =
-            "Projection: test.col_int32\n  Filter: CAST(test.col_int32 AS Float64) > __sq_1.__value\
-        \n    CrossJoin:\
-        \n      TableScan: test projection=[col_int32]\
-        \n      Projection: AVG(test.col_int32) AS __value, alias=__sq_1\
-        \n        Aggregate: groupBy=[[]], aggr=[[AVG(test.col_int32)]]\
-        \n          Filter: test.col_utf8 >= Utf8(\"2002-05-08\") AND test.col_utf8 <= Utf8(\"2002-05-13\")\
-        \n            TableScan: test projection=[col_int32, col_utf8]";
+        let expected = r#"Projection: test.col_int32
+  Filter: CAST(test.col_int32 AS Float64) > __sq_1.__value
+    CrossJoin:
+      TableScan: test projection=[col_int32]
+      SubqueryAlias: __sq_1
+        Projection: AVG(test.col_int32) AS __value
```

Comment:
Projections no longer have aliases, but use the generic `SubqueryAlias` operator instead